scripts/scripts_11.0/bootstrap.py

```diff
@@ -219,8 +219,8 @@ def init_seafile_server():
         src = join(topdir, fn)
         dst = join(shared_seafiledir, fn)
         if not exists(dst) and exists(src):
-            shutil.move(src, shared_seafiledir)
-            call('ln -sf ' + join(shared_seafiledir, fn) + ' ' + src)
+            call(['mv', '-f', src, dst])
+            call(['ln', '-sf', dst, src])
```

Comment:
IMHO the issue is not in `shutil` vs `call`.
What bugs me, however, is `ln` command syntax which looks wrong to me
```bash
# this is wrong:
ln -sf dst src

# this is right:
ln -sf src dst
```

Want to know a better way to memorize which arg goes first? Think of it as “the existing goes first”, as in:
```bash
ln -sf <existing> <symlink>
```